runtime/doc/diagnostic.txt

```diff
@@ -635,6 +588,38 @@ is_disabled({bufnr}, {namespace})               *vim.diagnostic.is_disabled()*
     Return: ~
         (boolean)
 
+jump({opts})                                           *vim.diagnostic.jump()*
+    Move to a diagnostic.
+
+    Parameters: ~
+      • {opts}  vim.diagnostic.JumpOptions A table with the following optional keys:
+                • diagnostic: (table) The diagnostic item to move to
+                  |diagnostic-structure|. Mutually exclusive with "count",
+                  "namespace", and "severity".
+                • count: (number) The number of diagnostics to move by. A
+                  positive integer moves forward by {count} diagnostics, while
+                  a negative integer moves backward by {count} diagnostics.
+                  Mutually exclusive with "diagnostic".
+                • namespace: (number) Only consider diagnostics from the given
+                  namespace. Only used when "count" is non-nil.
+                • cursor_position: (table) Cursor position as a (row, col)
+                  tuple. Used to find the nearest diagnostic when "count" is
+                  used. Only used when "count" is non-nil.
+                • wrap: (boolean) Whether to wrap around the file or not.
+                  Similar to 'wrapscan'. Default true.
+                • severity: Only consider diagnostics that match the given
+                  |diagnostic-severity|. Only used when "count" is non-nil.
+                • float: (boolean or table) If "true", call
```

Comment:
What is the default?

Reply:
Updated the docs to include default values/behaviors for all fields.